### DISCUSSION TITLE
How to get Markers to work for the Plot Visualization

### DISCUSSION THREAD

**Discussion by MMSlx** [nasa/openmct#8246] [Community Q&A]
Hello!

I've been trying to understand how to get markers to work. The core idea is to have a Telemetry source showing a timeseries of Data, and another source showing 'annotations' which would be rendered as markers. Am I understanding the 'marker' feature correctly?

Because right now, the problem is that the 'annotation' source has a text value, meaning the Plot cannot calculate a position for it. This suggests to me that either it is not meant to be used for this, or I am doing something wrong.

Could you tell me where I am going wrong and if what I am trying is at all possible?

**Comment by TylerURSA** [comment]
I'm also interested in learning how to get Markers to work in the Plot visualization. I've been unable to find anything relating to markers/annotations within the documentation. However, I have been able to find a few issues (like this one) regarding some kind of markers/alarm marker functionality on plots, but nothing regarding how to add this capability to my plots. Any help would be appreciated

**Comment by Kokodora** [comment]
I have the same issue and I also cant find anything in the Documentation. Any help would be appreciated.

**Comment by davetsay** [comment]
@MMSlx , @TylerURSA , @Kokodora , thanks for your patience and interest in Open MCT, and annotations in plots. I'm going to convert this to a discussion.
